store/CSDK/test/main.cpp

```diff
@@ -220,6 +393,86 @@ bool tryCatchException(JNIEnv *env) {
  */
 bool readFromS3(JNIEnv *env, char *argv[]) {
     printf("\nRead data from S3:\n");
+    struct timeval start, build, read;
```

Comment:
Like we discussed in the previous PR, separate S3 test cases are not required. only thing we do in s3 testcase is calling withHadoopConf API.  user can call this if he needs to run in S3 environment.

Reply:
ok, optimized, please check.